src/lib/get-settings/index.js

```diff
@@ -20,9 +20,8 @@ const mergeSettingsSources = (inputSettings = {}) => {
   return Object.assign({}, envSettings, inputSettings);
 };
 
-const getSettings = (inputSettings) => {
+const getSettings = (inputSettings, isUsingDeployUrl) => {
   const settings = mergeSettingsSources(inputSettings);
-  if (Object.keys(settings).length === 0) return;
```

Comment:
Now we always pass back the default config, which is the same as if no config is passed to lighthouse at all

Reply:
😊